modules/frontend/frontend.go

```diff
@@ -100,7 +99,8 @@ func NewTripperware(cfg Config, logger log.Logger, registerer prometheus.Registe
 
 			span.SetTag("response marshalling format", marshallingFormat)
 
-			traceID, _ := middleware.ExtractTraceID(ctx)
+			otelspan := trace.SpanFromContext(ctx)
```

Comment:
Is this backwards compatible when not using otel tracing?  Do we need to try to get a trace ID from both methods, or maybe main.go can setup a callback and provide it to this method somehow?

Reply:
Nope, it's not 🙃  This is still code from before I added the feature flag.

Reply:
Got this working now 🙂  I tried to test it but when you initialise one of the tracers they stay active for all the tests. I.e. the tests influence each other.